src/seedu/addressbook/AddressBook.java

```diff
@@ -812,6 +829,22 @@ private static boolean deletePersonFromAddressBook(HashMap<PersonProperty, Strin
     private static ArrayList<HashMap<PersonProperty, String>> getAllPersonsInAddressBook() {
         return ALL_PERSONS;
     }
+    
+    /**
+     * Returns all persons in the address book sorted alphabetically
+     */
+    private static ArrayList<HashMap<PersonProperty, String>> getAllPersonsInAddressBookSorted() {
+        TreeMap<String, HashMap<PersonProperty, String>> names = new TreeMap<String, HashMap<PersonProperty, String>>();
+        for (HashMap<PersonProperty, String> person : getAllPersonsInAddressBook()) {
+        	names.put(person.get(PersonProperty.NAME), person);
+        }
+        
+        ArrayList<HashMap<PersonProperty, String>> sortedList = new ArrayList<HashMap<PersonProperty, String>>();
+        for (HashMap<PersonProperty, String> sortedItem : names.values()) {
+        	sortedList.add(sortedItem);
```

Comment:
please use 4 spaces for indentation, as required by the coding standards.
Tip: you can get eclipse to do that for you, check out how!